components/cluster/command/root.go

```diff
@@ -78,6 +78,7 @@ func init() {
 	// Initialize the global variables
 	flags.ShowBacktrace = len(os.Getenv("TIUP_BACKTRACE")) > 0
 	cobra.EnableCommandSorting = false
+	gOpt.NativeSSH = len(os.Getenv(localdata.EnvNameNativeSSHClient)) > 0
```

Comment:
I don't think it's better to use `len(os.Getenv(localdata.EnvNameNativeSSHClient)) > 0` to indicate the native ssh has been used. e.g:
```
TIUP_NATIVE_SSH=false
or
TIUP_NATIVE_SSH=0
or
TIUP_NATIVE_SSH=disable
```

Prefer to `len(os.Getenv(localdata.EnvNameNativeSSHClient)) > 0 && ( os.Getenv(localdata.EnvNameNativeSSHClient)) == false || os.Getenv(localdata.EnvNameNativeSSHClient)) == 1 || os.Getenv(localdata.EnvNameNativeSSHClient)) == enable)`